Demo/app/src/main/java/com/krunal/demo/kotlinPractice/collection/CollectionMain.kt

```diff
@@ -0,0 +1,267 @@
+package com.krunal.demo.kotlinPractice.collection
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#naming-rules

we always go withe smaller case and word concatenation